tests/pytests/unit/utils/test_network.py

```diff
@@ -1,8 +1,1535 @@
+    assert interfaces == {
+        "lo0": {
+            "inet": [{"address": "127.0.0.1", "netmask": "255.0.0.0"}],
+            "inet6": [{"address": "fe80::1", "prefixlen": "64", "scope": "lo0"}],
+            "up": True,
+        },
+        "vioif0": {
+            "hwaddr": "00:a0:98:e6:83:18",
+            "inet": [
+                {
+                    "address": "192.168.1.80",
+                    "broadcast": "192.168.1.255",
+                    "netmask": "255.255.255.0",
+                }
+            ],
+            "inet6": [
+                {
+                    "address": "fe80::2a0:98ff:fee6:8318",
+                    "prefixlen": "64",
+                    "scope": "vioif0",
+                }
+            ],
+            "up": True,
+        },
+    }
+
+
+def test_freebsd_remotes_on():
+    with patch("salt.utils.platform.is_sunos", lambda: False):
+        with patch("salt.utils.platform.is_freebsd", lambda: True):
+            with patch("subprocess.check_output", return_value=FREEBSD_SOCKSTAT):
+                remotes = network._freebsd_remotes_on("4506", "remote")
+                assert remotes == {"127.0.0.1"}
+
+
+def test_freebsd_remotes_on_with_fat_pid():
+    with patch("salt.utils.platform.is_sunos", lambda: False):
+        with patch("salt.utils.platform.is_freebsd", lambda: True):
+            with patch(
+                "subprocess.check_output",
+                return_value=FREEBSD_SOCKSTAT_WITH_FAT_PID,
+            ):
+                remotes = network._freebsd_remotes_on("4506", "remote")
+                assert remotes == {"127.0.0.1"}
+
+
+def test_netlink_tool_remote_on_a():
+    with patch("salt.utils.platform.is_sunos", lambda: False):
+        with patch("salt.utils.platform.is_linux", lambda: True):
+            with patch("subprocess.check_output", return_value=LINUX_NETLINK_SS_OUTPUT):
+                remotes = network._netlink_tool_remote_on("4506", "local_port")
+                assert remotes == {"192.168.122.177", "::ffff:127.0.0.1"}
+
+
+def test_netlink_tool_remote_on_b():
+    with patch("subprocess.check_output", return_value=LINUX_NETLINK_SS_OUTPUT):
+        remotes = network._netlink_tool_remote_on("4505", "remote_port")
+        assert remotes == {"127.0.0.1", "::ffff:1.2.3.4"}
+
+
+def test_openbsd_remotes_on():
+    with patch("subprocess.check_output", return_value=OPENBSD_NETSTAT):
+        remotes = network._openbsd_remotes_on("4506", "remote")
+        assert remotes == {"127.0.0.1"}
+
+
+def test_openbsd_remotes_on_issue_61966():
+    """
+    Test that the command output is correctly converted to string before
+    treating it as such
+    """
+    with patch("subprocess.check_output", return_value=OPENBSD_NETSTAT.encode()):
+        remotes = network._openbsd_remotes_on("4506", "remote")
+        assert remotes == {"127.0.0.1"}
+
+
+def test_generate_minion_id_distinct():
+    """
+    Test if minion IDs are distinct in the pool.
+
+    :return:
```

Comment:
```suggestion
    :return:
```
What's up with this?
Doesn't look useful, so, remove it?